kod/object/passive/spell/radiusench/bane.kod

```diff
@@ -0,0 +1,116 @@
+% Meridian 59, Copyright 1994-2012 Andrew Kirmse and Chris Kirmse.
+% All rights reserved.
+%
+% This software is distributed under a license that is described in
+% the LICENSE file that accompanies it.
+%
+% Meridian is a registered trademark.
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+BaneEnchantment is RadiusEnchantment
+
+constants:
+
+   include blakston.khd
+
+resources:
+   
+   default_hurt_caster = "A tide of energy retaliates against your spell."
+   default_kills_you = \
+      "The oncoming tide of energy carries you straight to "
+      "the Underworld!"
+
+classvars:
+   
+   vrBaneHurt = default_hurt_caster
+   vrBaneKills = default_kills_you
+
+   vrIcon = QorBane_name_rsc
```

Comment:
Can't use a resource here from a file that hasn't been built yet. This is a dangerous error, because the buildserver won't pick it up (it uses 103's kodbase.txt) but any attempt to build without a kodbase.txt will fail here. I know we've discussed this to death already but this is why you absolutely MUST compile and test your code on your own, locally (including after an nmake clean) before submitting it. Sorry to sound patronising here, I just really can't stress enough how much of a serious issue this is. If I hadn't caught this now on the buildserver while testing the building of the new keybind (using nmake clean) this would have popped up when we were about to push the update and delayed it. I do want to include your pull requests in updates but I get nervous every time I add one to a milestone, and I feel like it would be really easy for you to solve that issue by just compiling them yourself before submitting, and doing some basic testing.